addons/pos_sale/models/pos_config.py

```diff
@@ -9,8 +9,13 @@ class PosConfig(models.Model):
     _inherit = 'pos.config'
 
     def _get_default_pos_team(self):
-        team = self.env['crm.team'].search(['|', ('company_id', '=', self.env.company.id), ('company_id', '=', False)], limit=1)
-        return team
+        team = self.env['crm.team'].search([('name', '=', 'Point of Sale'), '|', ('company_id', '=', self.env.company.id), ('company_id', '=', False)], limit=1)
```

Comment:
as the field is not required, no need to create sales team, just put the first as default.

Reply:
done

Reply:
it seems that sales team still created:  if team:
            return team
        return self.env['crm.team'].create({
            'name': 'Point of Sale',
            'active': True,
            })